src/linter/LinterContext.ts

```diff
@@ -94,7 +95,7 @@ export interface LintMetadata {
 	// TODO: Use this to store information shared across linters,
 	// such as the async flag state in manifest.json which might be relevant
 	// when parsing the Component.js
```

Comment:
The TODO can be removed. Maybe an update like this would make sense?

```suggestion
	// The metadata holds information to be shared across linters
```